examples/arithmetics/src/cli/cli-util.ts

```diff
@@ -5,10 +5,10 @@
  ******************************************************************************/
 
 import type { AstNode, LangiumDocument, LangiumServices } from 'langium';
-import fs from 'fs';
-import path from 'path';
+import vscodeUri from 'vscode-uri';
+import * as fs from 'node:fs';
```

Comment:
I feel like this is a matter of taste. I always perform `*` imports on node built-ins. I feel like the methods exposed by `fs` and `path` are otherwise pretty non-descriptibe - a `join` function can mean literally everything, whereas `path.join` is quite explicit.

Reply:
Alright, good pro argument.